v2/clients/http/utils/request.go

```diff
@@ -35,6 +37,36 @@ func GetRequest(ctx context.Context, returnValuePointer interface{}, baseUrl str
 	return nil
 }
 
+// GetRequestAndReturnBinaryRes makes the get request and return the binary response
```

Comment:
the go doc should state all the returned value, include the contentType

Reply:
Modified.